src/calculateRentalCost.js

```diff
@@ -28,7 +28,15 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  if (days < 3) {
+    return days * 40;
+  }
+
+  if (days < 7) {
+    return (days * 40) - 20;
+  }
+
+  return (days * 40) - 50;
 }
```

Comment:
please fix according [the checklist](https://github.com/mate-academy/js_task-transportation-on-vacation/blob/master/checklist.md) items 1 and 3